crates/trippy-tui/locales/app.yml

```diff
@@ -3,439 +3,549 @@ trippy:
   tr: "%{hop_count} atlanan ve %{flow_count} benzersiz %{plural_flows} keşfedildi"
+  pt: "descobriu %{hop_count} saltos e %{flow_count} %{plural_flows} únicos"
 unknown:
   en: "unknown"
   fr: "inconnu"
   tr: "bilinmeyen"
+  pt: "desconhecido"
 icmp:
   en: "icmp"
   fr: "icmp"
   tr: "icmp"
+  pt: "icmp"
 udp:
   en: "udp"
   fr: "udp"
   tr: "udp"
+  pt: "udp"
 tcp:
   en: "tcp"
   fr: "tcp"
   tr: "tcp"
+  pt: "tcp"
 status_failures:
   en: "%{failure_count} of %{total_probes} (%{failure_rate}%) probes failed"
   fr: "%{failure_count} sur %{total_probes} (%{failure_rate}%) sondes ont échoué"
   tr: "%{failure_count} / %{total_probes} (%{failure_rate}%) sondajın başarısız olması"
+  pt: "%{failure_count} de %{total_probes} (%{failure_rate}%) sondas falharam"
 status_failed:
   en: "Failed"
   fr: "Échec"
   tr: "Başarısız"
+  pt: "Falhou"
 status_running:
   en: "Running"
   fr: "En cours"
   tr: "Çalışıyor"
+  pt: "Executando"
 status_frozen:
   en: "Frozen"
   fr: "Gelé"
   tr: "Dondurulmuş"
+  pt: "Congelado"
 awaiting_data:
   en: "Awaiting data..."
   fr: "En attente de données..."
   tr: "Veri bekleniyor..."
+  pt: "Aguardando dados..."
 header_help:
   en: "elp"
   fr: "Aide"
   tr: "Yardım"
+  pt: "Ajuda"
 header_settings:
   en: "ettings"
   fr: "Paramètres"
   tr: "Ayarlar"
+  pt: "Configurações"
 header_quit:
   en: "uit"
   fr: "Quitter"
   tr: "Çıkış"
+  pt: "Sair"
 title_hops:
   en: "Hops"
   fr: "Sauts"
   tr: "Atlananlar"
+  pt: "Saltos"
 title_frequency:
   en: "Frequency"
   fr: "Fréquence"
   tr: "Sıklık"
+  pt: "Frequência"
 title_samples:
   en: "Samples"
   fr: "Échantillons"
   tr: "Örnekler"
+  pt: "Amostras"
 title_traces:
   en: "Traces"
   fr: "Traces"
   tr: "İzler"
+  pt: "Rastreios"
 title_flows:
   en: "Flows"
   fr: "Flux"
   tr: "Akışlar"
+  pt: "Fluxos"
 title_map:
   en: "Map"
   fr: "Carte"
   tr: "Harita"
+  pt: "Mapa"
 title_help:
   en: "Help"
   fr: "Aide"
   tr: "Yardım"
+  pt: "Ajuda"
 title_settings:
   en: "Settings"
   fr: "Paramètres"
   tr: "Ayarlar"
+  pt: "Configurações"
 bsod_failed:
   en: "Trippy Failed :("
   fr: "Trippy a échoué :("
   tr: "Trippy Başarısız :("
+  pt: "Trippy falhou :("
 bsod_quit:
   en: "Press q to quit"
   fr: "Appuyez sur q pour quitter"
   tr: "Çıkmak için q tuşuna basın"
+  pt: "Pressione q para sair"
 hop:
   en: "Hop"
   fr: "Saut"
   tr: "Atla"
+  pt: "Salto"
 rtt:
   en: "RTT"
   fr: "RTT"
   tr: "RTT"
+  pt: "RTT"
 title_chart:
   en: "Chart"
   fr: "Graphique"
   tr: "Grafik"
+  pt: "Gráfico"
 samples:
   en: "Samples"
   fr: "Échantillons"
   tr: "Örnekler"
+  pt: "Amostras"
 host:
   en: "Host"
   fr: "Hôte"
   tr: "Ana bilgisayar"
+  pt: "Host"
 no_response:
   en: "No response"
   fr: "Pas de réponse"
   tr: "Yanıt yok"
+  pt: "Sem resposta"
 dns_failed:
   en: "Failed"
   fr: "Échec"
   tr: "Başarısız"
+  pt: "Falhou"
 dns_timeout:
   en: "Timeout"
   fr: "Délai dépassé"
   tr: "Zaman aşımı"
+  pt: "Tempo esgotado"
```

Comment:
Is "Timeout" being used when something actually times out or to display a timeout (time limit)? If the former, then "Tempo esgotado" is correct, if the latter then "Limite de tempo" is better.

Reply:
It is the former; this will be shown if a time limit expires when trying to lookup the hostname associated with a given IP address.  So will keep it as-is.